commands/commands.go

```diff
@@ -334,6 +334,18 @@ func isCommandEnabled(cfg *config.Configuration, cmd string) bool {
 	return cfg.Os.Bool(fmt.Sprintf("GITLFS%sENABLED", strings.ToUpper(cmd)), false)
 }
 
+func requireGitVersion() {
+	minimumGit := "1.8.2"
```

Comment:
This is the absolute minimum requirement for Git LFS to run safely. We can add a const if something else needs this specific value, but nothing does right now. Other bits in the codebase that check the git version already use `IsGitVersionAtLeast()`:

```bash
$ ack IsGitVersionAtLeast
commands/command_clone.go
89:    	if !git.Config.IsGitVersionAtLeast("2.9.0") {

git/git.go
406:func (c *gitConfig) IsGitVersionAtLeast(ver string) bool {
790:   	if !Config.IsGitVersionAtLeast("2.8.0") {

git/git_test.go
205:   	if !Config.IsGitVersionAtLeast("2.5.0") {

lfs/hook.go
41:    	customHooksSupported := git.Config.IsGitVersionAtLeast("2.9.0")
```